tests/test_F0_output.py

```diff
@@ -213,7 +213,10 @@ def test_processing_dict_for_json_export_parse_numpy_int64(self):
     def test_processing_dict_for_json_export_parse_pandas_DatetimeIndex(self):
         """ """
         expr = F0.convert(JSON_TEST_DICTIONARY["pandas_DatetimeIndex"])
-        assert expr == "date_range"
+        assert (
+            expr
+            == '{"columns":[0],"index":[1577836800000,1577840400000,1577844000000],"data":[[1577836800000],[1577840400000],[1577844000000]]}'
+        )
```

Comment:
Is it necessary that index and data values are the same? Would it be possible to store the timestamps "in an array" but formatted as strings, ie.
"index": [1, 2, 3]
"data": ["2020-01-01 00:00", "2020-01-01 01:00", "2020-01-01 02:00"]?

However, we are not using this currently anyway, so we do not need to put much work into this. Lets leave it - as long as python is able to read the pd.datetimeindex from json now, there is no issue. Do we need to add a test to B0 to make sure that the format is readable?

Reply:
This is due to the formsting of the `to_json` function with `orient="split"` we could format it as you suggested, but then I am not sure we can form DatetimeIndex from it

Reply:
Okay. It is most important that the json is machine-readable, so it doesn´t matter.